src/components/molecules/Chatbox/Chatbox.scss

```diff
@@ -14,46 +14,71 @@
     overflow-x: auto;
   }
 
-  &__form {
-    background-color: $black;
-    height: 74px;
+  &__container {
     padding: 10px;
-    display: flex;
-    align-items: center;
+    background-color: $black;
+
+    & > *:first-child {
+      padding-bottom: 10px;
+    }
   }
 
-  &__input {
-    // @debt Workaround to avoid styles set by src/scss/global.scss (fix this when when clean that up)
-    all: unset;
-    outline: none;
-    background-color: $saturated-black;
-    flex: 1;
+  &__cancel-poll {
+    width: fit-content;
+    margin-left: auto;
+    text-decoration: underline;
     cursor: pointer;
-    font-size: 0.9em;
-    padding: 12px 12px;
-    border-radius: 28px;
-    color: $white;
+    svg {
+      margin-left: 5px;
+    }
   }
 
-  &__submit-button {
-    outline: none;
-    border: none;
-    min-width: 44px;
-    min-height: 44px;
-    border-radius: 50%;
+  &__dropdown.dropup {
     display: flex;
-    justify-content: center;
-    align-items: center;
-    margin-left: 10px;
+    justify-content: flex-end;
 
-    background-color: $primary;
+    .dropdown-toggle {
```

Comment:
Can we align these `.dropup`, `.dropdown-toggle`, etc classes to BEM naming standards?

- getbem.com/naming/

Reply:
.dropup, .dropdown-toggle are bootstrap classes which are used for overriding the default styles